app/javascript/controllers/index.js

```diff
@@ -4,8 +4,11 @@
 
 import { application } from "./application"
 
-import Clipboard from 'stimulus-clipboard'
-application.register('clipboard', Clipboard)
-
 import HelloController from "./hello_controller"
 application.register("hello", HelloController)
+
+import IntegrationController from "./integration_controller"
```

Comment:
couldn't find the file `integration_controller.js`, maybe we need to comment out these import

Reply:
I have fixed this error, ready for review @christinasugiono